packages/docs/content/docs/developer-updates/2025-12-25-weekly-update-16.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Weekly Update #16
+---
+
+```
+      *
+     / \
+    / 0 \
+   /_____\      // Merry Christmas
+      |         // May all your builds be green
+      |
+```
+
+# Weekly Update #16
+
+### TL;DR
+
+This release is still in progress, but here are some early previews of what’s coming.
+
+Also, starting this week, along with weekly updates, we’ll now include key debates and decisions happening inside the team. If you want a voice in what ships, do reply to us in the links given below.
+
+### 🔥 Preview
+
+- We're working on a feature that allows you to add, view, and manage inline comments directly in the editor when reviewing code changes.
+
+- You'll soon be able to choose the base branch when creating a new worktree, instead of being forced to use the current branch (`HEAD`). This makes it easier to start clean feature branches from main or any branch you prefer.
+
+### 🤓 Internal Tech Discussions
+
+- **Do AI coding agents review the wrong coding step?** We’re debating a shift in how coding agents should be reviewed. Most tools pause the workflow to approve a plan before execution. 
+
+   But in practice, plans happen before the agent has paid the cost of reality: before navigation, failed tests, dependency issues, and edge cases. The result is confident speculation, and not based on insights.
+
+   We believe that the real checkpoint should happen after the work: reviewing the walkthrough, the diffs, what broke, what got fixed, and why. That’s much closer to how engineers review code today. We’re debating this direction and would **[love your input](https://x.com/mzh1024/status/2002023627232092358)**.
+
+- **AI coding conversations should be append-only:** Instead of editing past prompts, the idea is to keep the history intact and fork from earlier points, similar to branching in Git. 
+
+  <video
+    controls
+    style={{
+      width: "100%",
+      borderRadius: "8px",
+      boxShadow: "0 4px 12px rgba(0, 0, 0, 0.15)",
+    }}
+   >
+    <source src="https://assets.docs.getpochi.com/forking.mp4" type="video/mp4" />
+    Your browser does not support the video tag.
+   </video>
+
+    Editing history might feel convenient, but it erases the reasoning, the failed attempts, and the learning that happened.
+    
+    **[Read the full post](https://x.com/mzh1024/status/2002748473511489909)** and tell us what you think!
+
+
+ ### 📖 Resources
```

Comment:
how about moving this short section above the discussions section since this is more related to tech announcements and easy to be overlooked